src/app/components/core/scheduler/scheduler.module.ts

```diff
@@ -38,7 +46,8 @@ const routes: Routes = [
     RouterModule.forChild(routes),
     NgProgressModule,
     PipesModule,
-    AlertDialogModule
+    AlertDialogModule,
+    SharedModule
```

Comment:
SharedModule is now being used here? and for?

Reply:
For `ViewConfigItemComponent`